irc_context.go

```diff
@@ -96,6 +97,52 @@ func (ic IrcContext) UserID() string {
 	return ic.User.ID
 }
 
+// GetChannels returns a list of channels the context is connected
+// to
+func (ic *IrcContext) GetChannels(refresh bool) []slack.Channel {
+	if refresh || ic.SlackChannels == nil || len(ic.SlackChannels) == 0 {
+		channels, err := ic.SlackClient.GetChannels(true)
```

Comment:
the `channels.list` method (used by `GetChannels`) is deprecated, https://api.slack.com/methods/channels.list . Can you replace it with a call to `GetConversations`? You also need to set the appropriate parameters, e.g.
```
params := slack.GetConversationsParameters{
	Types:  []string{"public_channel", "private_channel"},
	Cursor: nextCursor,
}
chans, nextCursor, err := ctx.SlackClient.GetConversations(&params)
```

See https://github.com/insomniacslk/irc-slack/blob/master/irc_server.go#L186 .